src/main/java/org/folio/oaipmh/helpers/AbstractGetRecordsHelper.java

```diff
@@ -130,7 +131,7 @@ public abstract class AbstractGetRecordsHelper extends AbstractHelper {
   private static final String FAILED_TO_ENRICH_SRS_RECORD_ERROR = "Failed to enrich srs record with inventory data, srs record id - %s. Reason - %s";
   private static final String SKIPPING_PROBLEMATIC_RECORD_MESSAGE = "Skipping problematic record due the conversion error. Source record id - {}.";
   private static final String FAILED_TO_CONVERT_SRS_RECORD_ERROR = "Error occurred while converting record to xml representation. {}.";
-  private static final String QUERY_TEMPLATE = "(source==FOLIO%s%s%s%s)";
+  private static final String QUERY_TEMPLATE = "(%s%s%s%s%s)";
```

Comment:
@obozhko-folio, please rename QUERY_TEMPLATE to be clear what kind of query is it.

Reply:
Done